opus/import/do_import.py

```diff
@@ -1693,13 +1695,39 @@ def get_pdsfile_rows_for_filespec(filespec, obs_general_id, opus_id, volume_id,
             pref = pref.upper()
         sort_order = pref + ('%03d' % sort_order_num)
         list_of_sublists = products[product_type]
+
+        skip_current_product_type = False
+
         for sublist in list_of_sublists:
+            if (impglobals.ARGUMENTS.import_use_row_files and
+                skip_current_product_type):
+                break;
             for file_num, file in enumerate(sublist):
                 version_number = sublist[0].version_rank
                 version_name = sublist[0].version_id
                 if version_name == '':
                     version_name = 'Current'
                 logical_path = file.logical_path
+
+                if (impglobals.ARGUMENTS.import_use_row_files and
+                    ('_summary.tab' in logical_path or
```

Comment:
It really seems like there should be a better way to do this rather than hard-coding index filename suffixes here but I'm not sure what it would be.